content/tracing/faq/i-want-to-instrument-a-language-you-don-t-yet-support-how-does-the-api-work.md

```diff
@@ -5,4 +5,4 @@ autotocdepth: 3
 customnav: tracingnav
 ---
 
-The API is open source and available on GitHub here: https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/api.py Documentation is coming but in the meantime please direct questions to our [help channels](/help).
\ No newline at end of file
+The API is open source and available on GitHub [here](https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/api.py).  Documentation is coming but in the meantime please direct questions to our [help channels](/help).
```

Comment:
I dont think the API is open source. The agent and receiver are?

@jeremy-lq can you help with wording here?